src/fuzz.zig

```diff
@@ -22,8 +31,12 @@ pub fn main() !void {
     const maybe_filter = cli_args.next();
     const filter = blk: {
```

Comment:
A worthwhile safety improvement that just occurred to me here would be that it'd be nice to move `maybe_filter` into the `blk` block here, as the return value of `.next()` is only valid up until its next call - technically on posix systems this ultimately has no impact since its actually a pointer to `std.os.argv` I believe, but if we're using the iterator API we might as well use it right. Same idea wrt `maybe_seed`.